crates/apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -1932,41 +1908,35 @@ mod test_finalize_block {
         // Advance to the start of epoch 1. Val1 is the only block proposer for
         // the rest of the epoch. Val4 does not vote for the rest of the epoch.
         let height_of_next_epoch =
-            shell.wl_storage.storage.next_epoch_min_start_height;
+            shell.state.in_mem().next_epoch_min_start_height;
         let current_height = 4_u64;
-        assert_eq!(current_height, shell.wl_storage.storage.block.height.0);
+        assert_eq!(current_height, shell.state.in_mem().block.height.0);
 
         for _ in current_height..height_of_next_epoch.0 + 2 {
-            dbg!(
-                get_rewards_acc(&shell.wl_storage),
-                get_rewards_sum(&shell.wl_storage),
-            );
+            // dbg!(get_rewards_acc(&shell.state),
```

Comment:
Don't we need this dbg anymore?

Reply:
we shouldn't need these unless the tests are failing but I'll put them back to leave them as they are

Reply:
done in 43bc13e61b40c67fb35b552e170b2f77d93918c6